hmp/transform.py

```diff
@@ -102,7 +101,7 @@ def user_input_n_comp(data):
 # TODO: n_comp  None value input
 # TODO: printing to proper logging
 
-class DataTransformer:
+class Preprocessing:
```

Comment:
Proposed new names, transform.py -> preprocessing.py (not yet implemented), DataTransformer -> Preprocessing

Reply:
I suggest we do that as a separate PR?